processor/src/test/java/org/mapstruct/ap/internal/util/StringsTest.java

```diff
@@ -19,16 +19,21 @@
 package org.mapstruct.ap.internal.util;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Locale;
 
 import org.junit.Test;
 
 /**
  * @author Filip Hrisafov
  */
 public class StringsTest {
+
+    private static Locale TURKEY_LOCALE = getTurkeyLocale();
```

Comment:
This should be `private static final` that is one of the reasons why the build fails.